packages/core/src/transport/eventBridge.spec.ts

```diff
@@ -33,3 +33,28 @@ describe('isEventBridgePresent', () => {
     })
   })
 })
+
+describe('getEventBridge', () => {
+  let eventBridgeStub: DatadogEventBridge
+
+  beforeEach(() => {
+    updateExperimentalFeatures(['event-bridge'])
+    eventBridgeStub = initDatadogEventBridgeStub()
+  })
+
+  afterEach(() => {
+    resetExperimentalFeatures()
+    deleteDatadogEventBridgeStub()
+  })
+
+  it('should serialize the event for the datadog bridge', () => {
```

Comment:
instead of: 
`getEventBridge should serialize the event for the datadog bridge`
what about:
`event bridge should serialize sent events`